src/mnd/commands/deploy.cr

```diff
@@ -0,0 +1,97 @@
+require "../api/buildkite"
+
+module Mnd
+  class Commands::Deploy < Commands::Base
+    summary "Deploy app to a server"
+    <<-EOF
+    mnd deploy # deploy current branch to a server (chosen interactively)
+    EOF
+
+    def perform
+      pipeline_slug = get_and_verify_deploy_pipeline_slug_from_path
+      branch = Mnd::Utils::Git.current_branch
+
+      Mnd::Utils::Git.verify_remote_branch_up_to_date!(branch)
+
+      display.info "Deploying the '#{branch}' branch..."
+
+      available_deployment_targets.each_with_index do |target, i|
+        display.info "#{i + 1}. #{target["label"]}"
+      end
+
+      if available_deployment_targets.size == 1
+        deployment_target = available_deployment_targets[0]
+      else
+        print "Where would you like to deploy? [1..#{available_deployment_targets.size}] "
+
+        answer = gets.to_s.to_i
```

Comment:
Personally, I'd prefer a interface like `mnd deploy funnel` or whatever, with the available choices printed if the given choice isn't correct

Reply:
That's what I had in mind to begin with but wasn't sure whether it was actually worth adding the extra argument parsing and making it possible to mis-spell stuff. Since you bring it up I'll add it though 👍